packages/devtools-evm-hardhat/src/simulation/assets/nginx.conf

```diff
@@ -0,0 +1,56 @@
+events {}
+
+http {
+  # We will modify the log  format to include the target_network
+  log_format unique '$remote_addr - $remote_user [$time_local] '
+                    '"$request" $status $body_bytes_sent '
+                    '"$http_referer" "$http_user_agent" '
+                    'Network: "$target_network"';
+
+  server {
+    # This proxy server will listen on port 8545
+    # 
+    # Even though it's not ideal to have this hardcoded, this port
+    # will be remapped to a desired host port using docker compose,
+    # the only issue this hardcoding brings is the fact that this port
+    # needs to match the container port in the compose spec
+    listen 8545;
+    listen [::]:8545;
+
+    # We will add a simple endpoint for healthcheck
+    location /health-check {
+      access_log	off;
+      error_log	off;
+      return 200 'ok';
+    }
+
+    # In this section we'll proxy all the requests to this server
+    # to the respectative network nodes
+    # 
+    # The requests are proxied based on the first path segment:
+    # 
+    # http://localhost/fuji -> http://fuji:8545/
+    # 
+    # For now the remaining path segments are not being preserved:
```

Comment:
This might be a gotcha thats going to be hard to debug, it does not look hard to capture the remaining path in the regex, what do you think about preserving it?

Reply:
For this particular use case, there's not much point since the only endpoint on the RPC we are interested in is `/` - that's where all the requests are being sent. So not much value for the added complexity